ci-operator/step-registry/cucushift/installer/rehearse/ibmcloud/ipi/with-ep-cos/provision/cucushift-installer-rehearse-ibmcloud-ipi-with-ep-cos-provision-chain.yaml

```diff
@@ -0,0 +1,32 @@
+chain:
+  as: cucushift-installer-rehearse-ibmcloud-ipi-with-ep-cos-provision
+  steps:
+  - chain: ipi-conf-ibmcloud
+  - ref: ipi-install-monitoringpvc
+  - ref: ipi-conf-ibmcloud-custom-endpoints
+  - chain: ipi-install
+  - ref: enable-qe-catalogsource
+  - chain: cucushift-installer-check
+  - ref: cucushift-installer-check-ibmcloud
+  - ref: cucushift-installer-check-ibmcloud-custom-endpoints
+  env:
+  - name: BASE_DOMAIN
+    default: "ibmcloud.qe.devcluster.openshift.com"
+  - name: SERVICE_ENDPOINT_IAM
+    default: ""
+  - name: SERVICE_ENDPOINT_VPC
+    default: ""
+  - name: SERVICE_ENDPOINT_ResourceController
+    default: ""
+  - name: SERVICE_ENDPOINT_ResourceManager
+    default: ""
+  - name: SERVICE_ENDPOINT_DNSServices
+    default: ""
+  - name: SERVICE_ENDPOINT_COS
+    default: "DEFAULT_ENDPOINT"
```

Comment:
I'm not sure if we want to rename the `"DEFAULT_ENDPOINT"` to `"DEFAULT_PRIVATE_ENDPOINT"` in those scripts, actually I'm a bit confused to see the "DEFAULT_ENDPOINT" word in the first, it sounds like it's meaning the default public  endpoint being used by a basic IPI install, but it should be the default regional private endpoint.

Reply:
The most of default endpoints (except COS is using direct endpoint) we defined indeed is private endpoints, but I am not sure if there would be some other change in the future, to changed them to other custom endpoints, another point is to keep consistent with aws custom endpoint with the same keyword.

Reply:
OK, sounds like maybe `DEFAULT_CUSTOM_ENDPOINT` would be more appropriate. If AWS custom endpoint is already using this value, let's keep it as now then.